Swin-Transformer/Swin-Transformer.patch

```diff
@@ -84,7 +93,12 @@ index 84230ea..faa9291 100644
  
      args, unparsed = parser.parse_known_args()
  
-@@ -95,9 +106,22 @@ def main(config):
++    args.local_rank = int(os.environ["LOCAL_RANK"])
```

Comment:
It is more compatible with args.local_rank and env LOCAL_RANK.
```python
    if args.local_rank is None and 'LOCAL_RANK' in os.environ:
        args.local_rank = int(os.environ['LOCAL_RANK'])
```

Reply:
Fixed.